python/ray/data/read_api.py

```diff
@@ -105,6 +108,8 @@ def from_items(items: List[Any], *, parallelism: int = -1) -> Dataset[Any]:
         items: List of local Python objects.
         parallelism: The amount of parallelism to use for the dataset.
             Parallelism may be limited by the number of items.
+        output_arrow_format: If True, returns data in Arrow format, instead of Python
+            list format. This only works if items contains a list of Defaults to False.
```

Comment:
`This only works if items contains a list of` does not end.

Reply:
good catch, updated